fractions/microprofile/microprofile-health/src/main/java/org/wildfly/swarm/monitor/HealthMetaData.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2015-2017 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.swarm.monitor;
```

Comment:
I know these are directly copied from monitor fraction, and that when we switch from usage of monitor to microprofile-health the packages being the same makes that easier.

But I'm still wondering if we shouldn't rename the package to use `health` instead of `monitor`

Reply:
That is fine, I'll change the package to health.